lib/classes/Service.js

```diff
@@ -33,6 +33,8 @@ class Service {
   load(opts) {
     const that = this;
     const options = opts || {};
+    options.stage = options.stage || options.s;
```

Comment:
Agreed. That's why I thought it's strange and they should be processed right away. But the thing is loading happens before anything for validation.